### PR TITLE
add su support introduced in logrotate 3.8.0

### PR DESCRIPTION
**Pull request by CharlieR-o-o-t**

### CODE REVIEW

File: manifests/rule.pp [hunk 1]
```diff
@@ -337,6 +349,14 @@
     }
   }
 
+  case $su {
+    'undef',false: {}
+    true: { $_su = 'su' }
```
Comment (alvagante) [inline manifests/rule.pp]: Variables names starting with _ are depracated and probably fail on Puppet 4

File: templates/rule.erb [hunk 2]
```diff
@@ -18,6 +18,17 @@
     end
   end
 
+  if scope.to_hash.has_key?('_su')
```
Comment (alvagante) [inline templates/rule.erb]: This part looks a bit overkill, why not just doing something like:
<% if @su %>
su <%= @su_owner %>  <%= @su_group %> 
<% end>
without the need to set a @_su variable.
(am I missing something?)

**Comment by CharlieR-o-o-t** [comment]
yes, I agree, it's overhead, I just merged realization from logrotate/rodjek. 

I have fixed everything what was in discussion and updated commit. 
Thanks

**Comment by alvagante** [comment]
thx

**Comment by CharlieR-o-o-t** [comment]
Sorry, I had mistype at rule.erb :(((

**Pull request merged**